docs/examples/form/component.py

```diff
@@ -5,8 +5,6 @@
 
 @register("form")
 class Form(Component):
-    template_file = "form.html"
```

Comment:
Because the examples are defined in `docs/examples`, this was causing problem when I tried to test the components if they at the same time defined separate HTML or CSS files. I didn't want to have to think how to configure the template directories correctly. So instead I inlined the HTML/JS/CSS inside Python.